lib/getStagedFiles.js

```diff
@@ -31,8 +31,8 @@ export const getStagedFiles = async ({ cwd = process.cwd(), diff, diffFilter } =
      * roots and get the filename.
      */
     return output
-      .split(':')
       .slice(1)
+      .split('\u0000:')
```

Comment:
Should we first `split` and then `splice` like before? Doesn't it now just remove the first character?

Reply:
If we split first then we might need a `map` to remove the first colon.

```js
return output
  .split('\u0000:')
  .map(entry => entry.startsWith('.') ? entry.slice(1) : entry)
```

So to save the `map` computation cost, we can slice away the first colon then do the split.

Reply:
Makes sense!